lib/librobdb/robdb.spec

```diff
@@ -0,0 +1,64 @@
+Name:           robdb
+Version:        1.1
+Release:        %{autorelease -n %{?dist}}
+Summary:        Provide basic functions to search and read Berkeley Database records
+
+License:        GPL-2.0-or-later
+URL:            https://github.com/389ds/389-ds-base/lib/librobdb
```

Comment:
Broken URL

Reply:
That is expected:
Right now, lib/librobdb does not exists yet on main branch, but it will exist once the PR get merged.

Reply:
Then it would be something like this (note the `tree` and `main` branch in the URL):
https://github.com/389ds/389-ds-base/tree/main/lib/librobdb